src/styles/theme/oc-datepicker.scss

```diff
@@ -99,10 +99,11 @@
   &-popup {
     @extend .uk-position-fixed;
     @extend .uk-position-center;
-    @extend .uk-position-z-index;
     @extend .uk-box-shadow-large;
     @extend .uk-background-default;
 
+    z-index: 9000;
```

Comment:
There is a `$z-index-modal` scss variable. Should be usable here. Could you try?